src/sentry/monitors/logic/mark_failed.py

```diff
@@ -196,24 +185,6 @@ def mark_failed_threshold(
     return True
 
 
-def mark_failed_no_threshold(failed_checkin: MonitorCheckIn):
-    from sentry.signals import monitor_environment_failed
-
-    monitor_env = failed_checkin.monitor_environment
-
-    monitor_env.update(status=MonitorStatus.ERROR)
-
-    # Do not create event if monitor or monitor environment is muted
-    if monitor_env.monitor.is_muted or monitor_env.is_muted:
-        return True
-
-    create_legacy_event(failed_checkin)
```

Comment:
Could you also remove this function and any other unused functions that it might call?